DTR_futbol.md

```diff
@@ -0,0 +1,77 @@
+Project: Futbol
+
+Group Member Names: Jake Heft, Joshua Carey, Hope Gochnour, Will Dunlap
+
+Goals and Expectations for the Project (What does each group member hope to get out of this project? What do we want to achieve as a team? How will we know that we're successful?):
+
+- Jake: Get better at planning a project (grow in project organization skills) and refine GH workflow.
+- Hope: Get better at handling how files are read from and how to work in a group.
+- Joshua: Get better at approaching project with big picture in mind.
+- Will: Learn effective ways of how to break up the project that saves time and makes coding more efficient. Growing in articulating thoughts/ideas before execution.
+- Project Driver: getting better and learning new stuff.
+
+Team strengths & collaboration styles (consider discussing your Pairin qualities here):
+
+- Jake: Would like to see a mix of both individual and collaborative coding.
+- Joshua: Preference towards independent work in terms of creativity.
+- Will: Most efficient as independent worker and with intermittent breaks.
```

Comment:
Update : "Will: has had some success as an independent worker, but appreciates open dialog and talking through problems/challenges with other people.  Collaboration has generally led to better outcomes.